src/processout/paymenttoken.ts

```diff
@@ -0,0 +1,48 @@
+/// <reference path="../references.ts" />
+
+/**
+ * ProcessOut module/namespace
+ */
+ module ProcessOut {
+    /**
+     * PaymentToken is the class handling a Payment token payload
+     */
+    export class PaymentToken {
+       /**
+         * Payload sent along when creating the ProcessOut Payment token payment.
+         * @type {any}
+         */
+        protected payload: any;
```

Comment:
It's almost never preferable to use `any` unless there is a good reason to (for example, working with libraries that lack proper typing). There is a great article explaining this in detail here:

https://thoughtbot.com/blog/typescript-stop-using-any-there-s-a-type-for-that

I think a better approach would be to use `unknown` here. That way you can still store any (non-null) data type in this `payload` property, but the compiler will be more strict about how you can use it (most notably, it won't let you assign it to anything or access methods from it, since these would cause run-time errors).